src/BenchmarkDotNet/Extensions/ProcessExtensions.cs

```diff
@@ -132,6 +134,17 @@ internal static void SetEnvironmentVariables(this ProcessStartInfo start, Benchm
             if (!benchmarkCase.Job.HasValue(EnvironmentMode.EnvironmentVariablesCharacteristic))
                 return;
 
+            if (benchmarkCase.Job.Infrastructure.Toolchain is MonoAotLLVMToolChain)
```

Comment:
I'm new here :) Can we add a way to specify these kinda environment variables, maybe in the toolchain, or the runtime? and `ProcessExtensions` can call a method to get the variables

Reply:
I think that is a good idea for an additional feature. But for now I just want to do this, because we pretty much need this environment variable set to actually use llvm, which is the expected behavior here.